crawler.js

```diff
@@ -437,15 +463,28 @@ class Crawler {
       console.warn("Link Extraction failed", e);
       return;
     }
-
     this.queueUrls(results);
+
+    try {
+      var filePath = path.join("collections/", this.params.collection, 'archive/pages/pages.jsonl')
```

Comment:
small nitpick: just reuse dirPath
```suggestion
      var filePath = path.join(dirPath, 'pages.jsonl')
```